dvc/command/commit.py

```diff
@@ -47,7 +47,7 @@ def add_parser(subparsers, parent_parser):
         "--force",
         action="store_true",
         default=False,
-        help="Commit even if checksums for dependencies/outputs changed.",
+        help="Commit even if hash value for dependencies/outputs changed.",
```

Comment:
@jorgeorpinel , missed the conversation when we decided that _hash value_ was better than _checksums_ :sweat_smile:

Reply:
@jorgeorpinel let's keep it for DVC itself for now. It should be a separate discussion/PR to fix it.

@mroutis technically, checksum is not a right term for us. md5 (sha1, etc) is hash (that can be used to check for integrity - that's what checksums are for). But the most important here that we rely on it being a good _hash_. We need uniqueness. That's why for example HDFS checksums do not work for us for external outputs/inputs.

Reply:
@shcheklein , I agree that _checksum_ is not the best term, indeed) just caught for surprise.

Reply:
it would be great to have an issue for this, tho

Reply:
> missed the conversation when we decided that hash value was better than checksums

Yeah sorry it happened somewhere in iterative/dvc.org/pull/933 but the discussion is now resolved/hidden since it's such a long PR. Ivan brought up that cheksums have the intention to check the validity of files and that's not the main reason we use them in DVC... Technically the MD5 string is a hash or "hash value" (more general concept).

There's no issue but there's a (draft) PR for this in iterative/dvc.org/pull/962 for now.

Reply:
I think there is no reason do this ad-hoc in the DVC core repo. We should create a separate issue and deal with it "at-once".

Reply:
Yes, that's what iterative/dvc.org/pull/962 is for 🙂 but are you saying I should revert the few ones I changed here?

Reply:
yes, that's the idea to keep PRs focused as much as possible. Make it longer to review and merge when there are many things (especially something that requires discussions) happening at once. I would revert this for now.